qa/os/src/test/java/org/elasticsearch/packaging/test/DockerTests.java

```diff
@@ -225,19 +232,23 @@ public void test042KeystorePermissionsAreCorrect() throws Exception {
      * is minimally functional.
      */
     public void test050BasicApiTests() throws Exception {
-        waitForElasticsearch(installation, USERNAME, PASSWORD);
+        waitForElasticsearch(installation, USERNAME, PASSWORD, ServerUtils.getCaCert(installation));
```

Comment:
You are right. Initially , I was copying the cert from the container after `runContainer` in test10 above and then passed it around in all the `waitForElasticsearch` calls. Since I moved this logic to `ServerUtils.getCacert` we don't need it at all here